### ISSUE TITLE
 Ultimate Antibot addon

### ISSUE THREAD

**Issue by bedikee16** [plan-player-analytics/Plan#3280]
### Data to display

I want Ultimate AntiBot on my site! Possible?

### Link to API or Source Code

https://github.com/Kr1S-D/UltimateAntibotRecoded

### Link to Project page or Downloads

https://www.paypal.com/donate/?hosted_button_id=KKW3QJTCHPJZY

**Comment by AuroraLS3** [comment]
I don't know these plug-ins, you gotta tell me what sort of data there is that you want to see

**Comment by bedikee16** [comment]
whitelist people, blacklist people, you could see these or something else

**Comment by bedikee16** [comment]
> I don't know these plug-ins, you gotta tell me what sort of data there is that you want to see
can you solve it?

**Comment by AuroraLS3** [comment]
Yeah that's good. I assume you meant 'understand' instead of 'solve'

**Comment by bedikee16** [comment]
Yeah, i speak hungarian
